### DISCUSSION TITLE
Start the artisan modbus serial RTU for the first time, Serial Exception: Unable to open serial port.

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1673] [General]
Restart the communication is normal but there is an error. How to set it?

**Comment by Terracotta-6** [comment]
https://github.com/user-attachments/assets/a826bdca-0f33-4f1c-9ace-71e40ab015c1

**Comment by Terracotta-6** [comment]
If the artisan is not restarted, reinsert the USB port to the RS485 port and connect via MODBUS with the artisan port.

**Comment by MAKOMO** [comment]
What is your MODBUS timeout set to? What happens if you increase this one somewhat?

**Reply by Terracotta-6** [reply]
Reduce the hardware communication rate to 38400 for more stability.
The wrong Serial Command was set in Evnts.  Sorry to trouble you!
